src/webgpu/api/validation/encoding/cmds/buffer_texture_copies.spec.ts

```diff
@@ -1,10 +1,6 @@
 export const description = `
 copyTextureToBuffer and copyBufferToTexture validation tests not covered by
 the general image_copy tests, or by destroyed,*.
-
-TODO:
-Add tests to cover the validation rule that source.offset is a multiple of the texel block size of
-destination.texture.[[format]].
```

Comment:
I removed this comment because I found we have already had the validation tests on the copy buffer offset with all color formats, where all the depth/stencil tests are not tested (filtered by ```formatCopyableWithMethod```)

https://github.com/gpuweb/cts/blob/main/src/webgpu/api/validation/image_copy/layout_related.spec.ts#L199

Reply:
Ah, I see. Shouldn't all of the copy buffer offset tests be in this file, instead of `image_copy/layout_related`? Because the offset restrictions apply only to B2T/T2B but not writeTexture.

Reply:
Or perhaps the new tests in this file should move to that file, so they can, like L199, test that writeTexture does not have the restriction.

Reply:
Oh I separated all the tests about depth/stencil tests from that file into this file just for simplicity (they are quite 'special' as we need to take care about 'aspects').

I can either merge all the tests here into image_copy/layout_related.spec.ts in this PR or in the next one (after landing this PR first). What do you think?

Reply:
They don't need to be actually merged, I just think it would make sense for them to live near each other in the same file so it's easy to find them. Either is OK with me.

Reply:
Got it.

In my next PR I'd like to move all the tests from this file to the proper places in image_copy/ to make full use of the infrastructure there and test writeTexture() with depth/stencil formats. What do you think?

Reply:
Sounds great, thanks!